pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -656,6 +656,7 @@ async def run(self):
                             for pkg in packages.values():
                                 package = Package(**Package.createrepo_to_dict(pkg))
                                 artifact = Artifact(size=package.size_package)
+                                checksums["package_checksum_type"] = package.checksum_type
```

Comment:
Saving original `package_checksum_type` to use it as default when publishing

Reply:
there is not need to iterate through all the packages here this is similar to https://github.com/pulp/pulp_rpm/pull/1655#discussion_r408936878

in addition, on the Package model we already have checksum_type that is used in case no checksum_type is specified https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/models/package.py#L357

I do not think we need "package_checksum_type" at all